### ISSUE TITLE
SQL Injections

### ISSUE THREAD

**Issue by JRyuzaki** [WH0SUMIT/Artistry#1]
I found some SQL injection vulnerabilities in your project. Here is one example: https://github.com/WH0SUMIT/Artistry/blob/b7884f46a0e33d18a381a703d3303380d997229a/login.php#L11
In order to fix it simply switch to using prepared statements.

**Comment by WH0SUMIT** [comment]
This is a sample project, not the original one, but thank you for noticing this issue, any suggestions from your side ??

**Comment by JRyuzaki** [comment]
Yes as I said use prepared statements like shown here https://www.w3schools.com/php/php_mysql_prepared_statements.asp
Also can you link me to the original project? :smile:
